src/main/java/ua/softserve/rv_028/issuecitymonitor/controller/PDF/PdfWritable.java

```diff
@@ -0,0 +1,27 @@
+package ua.softserve.rv_028.issuecitymonitor.controller.PDF;
+
+import ua.softserve.rv_028.issuecitymonitor.entity.User;
+
+import java.time.LocalDateTime;
+
+public interface PdfWritable {
+    Long getId();
```

Comment:
Maybe better use `default ` parameter for method? 
And you could override only those that you want. 
```
public interface Test {
    default Long getId(){
        return null;
    }
    
    default String getTitle(){
        return null;
    }
    
    default String getCat(){
        return null;
    }    
}
```

Reply:
Done!